### DISCUSSION TITLE
Set a landing gear servo at reaching a waypoint

### DISCUSSION THREAD

**Discussion by techdesk786** [iNavFlight/inav#9649] [General]
I would like to know how to add a servo trigger for landing gear upon reaching a waypoint in INAV 7. I searched the internet extensively but cannot find any help

**Comment by MrD-RC** [comment]
You can use the programming framework yo set a condition when a waypoint is reached. 

Check out the programming [framework documentation.](https://github.com/iNavFlight/inav/blob/master/docs/Programming%20Framework.md) Also this section of video may point you in the right direction. What you want to do is not as complex as what’s shown in the video https://youtu.be/mLSva0UFcNI?t=2794&si=ru_p4KuRtkRqFz_8
